### PR TITLE
Improve front page

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-ccb 

This updates our website to direct people to github discussions for help. Once this is merged I will send a note to the mailing list indicating the change.

I need suggestions for what items to put in the list of features. I think it should be smaller than we have now and every item should link to something. Please add a comment with the content and markdown file of things we should list.

(refs #13580)

### CODE REVIEW

**Comment by dschwen** [comment]
On mobile the videos do not play.

**Comment by aeslaughter** [comment]
> On mobile the videos do not play.

Is that true for the gallery as well? And can you help figure out why? 

Also, do you know why Chrome and Safari show the background color as (32,32,32) even though the CSS says (34,34,34)? Firefox does it correct. At least for me.

**Comment by dschwen** [comment]
On the gallery page they autoplay and have controls when I tap them, on the
front page neither. I wonder if it is the dual video light/dark hack?

**Comment by aeslaughter** [comment]
> I wonder if it is the dual video light/dark hack?

Could be, do you have a better solution? Can you whip up a little JS for it?

**Comment by milljm** [comment]
At line 1162 in moose.css so the links look better in Safari
```css
 a {
   color: var(--links-dark);
   font-weight: 400;
   font-size: 100%;
 }
```
I wouldn't mind contributing to this, but I don't want to step on toes.

**Comment by aeslaughter** [comment]
@milljm You can add to this, just add a commit.

**Comment by aeslaughter** [comment]
@idaholab/moose-team Does the video play and have the correct background?
https://mooseframework.inl.gov/docs/PRs/15994/site

**Comment by cticenhour** [comment]
Here's what the gif and training video looks like on my machine. Everything plays as expected.

Screen Shot 2020-10-27 at 6 40 15 PM

File: framework/doc/content/js/init.js [hunk 1]
```diff
@@ -13,11 +13,13 @@
...
-   //   onClose: function(el) {var icons = $(el).find('i'); $(icons[0]).text('keyboard_arrow_down');}
-      // });
-
-  }); // end of document ready
+    // Change video source for dark mode, see media.py RenderVideo
+    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        $('.moose-video source[data-dark-src]').each(function(){
+            var src = $(this).data('dark-src');
+            $(this).attr('src', src);
+        });
```
Comment (aeslaughter) [inline framework/doc/content/js/init.js:23]: @dschwen Any idea why this would work locally with a build --serve but not on the actual server?

**Comment by cticenhour** [comment]
In Safari, there is a dark, watermarked background.

image

**Comment by aeslaughter** [comment]
@cticenhour Your are the only one to look at this, if you think it is reasonable let merge it. We can always add more in future and work on the video loading.

**Review by loganharbour — APPROVED** [review]
I looked, too!

**Review by cticenhour — COMMENTED** [review]
@aeslaughter Sorry for the delay -- didn't see this in my email immediately, but I would like to make one comment even if its already been merged. I wish we could make that intermittent gif selection bug for light/dark a bit more stable for things like Firefox, but that's something that could be improved in the future. No sense in holding this up for that. I like it overall.

**Pull request merged**